docs/posts/overview-of-ballerina-0.982.0.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Overview of Ballerina 0.982.0
+author: Maryam Ziyad Mohamed
+date: 01 October 2018
+status: Published
+abstract: Introduction to New Features and Changes introduced with Ballerina 0.982.0
```

Comment:
If you use title case here, 'introduced' must also start with an uppercase letter.